.github/workflows/Lint.yml

```diff
@@ -0,0 +1,29 @@
+name: Lint
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Python Setup
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Create Environment
+      shell: bash
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -e '.[lint]'
```

Comment:
is this installing black, isort, autoflake from the `lint` section of setup.py?

Reply:
Yes.